sql/moz-fx-data-shared-prod/telemetry/clients_last_seen_v1/view.sql

```diff
@@ -61,6 +66,46 @@ SELECT
       ) <= 1
     )
   ) AS is_allweek_regular_v1,
+  BIT_COUNT(days_visited_1_uri_bits) >= 21 AS is_core_active_v1,
+  CASE
+  WHEN
+    BIT_COUNT(days_visited_1_uri_bits)
+    BETWEEN 1
+    AND 6
+  THEN
+    'infrequent_user'
+  WHEN
+    BIT_COUNT(days_visited_1_uri_bits)
+    BETWEEN 7
+    AND 13
+  THEN
+    'casual_user'
+  WHEN
+    BIT_COUNT(days_visited_1_uri_bits)
+    BETWEEN 14
+    AND 20
+  THEN
+    'regular_user'
+  WHEN
+    BIT_COUNT(days_visited_1_uri_bits) >= 21
+  THEN
+    'core_user'
+  ELSE
+    'other'
+  END
+  AS activity_segments_v1,
+  days_since_created_profile = 6
+  AND BIT_COUNT(
+    days_seen_bits & udf.bits28_from_string('0000000000000000000001111111')
```

Comment:
We should probably convert these UDF calls to integer literals in hex notation to avoid the UDF overhead, and add commentary about the UDF equivalent.